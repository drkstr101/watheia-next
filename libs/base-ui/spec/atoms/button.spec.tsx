import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import Button from '../../src/lib/atoms/button';

describe('Button', () => {
  it('should render successfully', () => {
    const { container } = render(<Button />);
    expect(container).toMatchSnapshot();
  });
});
