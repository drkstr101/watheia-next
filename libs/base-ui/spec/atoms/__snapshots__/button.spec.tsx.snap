// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Button > should render successfully 1`] = `
<body>
  <div>
    <button
      class="react-aria-Button"
      data-rac=""
      type="button"
    />
  </div>
</body>
`;
